tests/util.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"io/ioutil"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var KubeVirtOcPath = ""
+
+func ProcessTemplateWithParameters(srcFilePath, dstFilePath string, params ...string) string {
+	By(fmt.Sprintf("Overriding template from %s to %s", srcFilePath, dstFilePath))
+	args := []string{"process", "-f", srcFilePath}
+	args = append(args, params...)
+	out, err := ktests.RunOcCommand(args...)
+	Expect(err).ToNot(HaveOccurred())
+	filePath, err := writeJson(dstFilePath, out)
+	Expect(err).ToNot(HaveOccurred())
+	return filePath
+}
+
+func CreateResourceWithFilePath(resourceName, resourceType, filePath string) {
+	By(fmt.Sprintf("Creating %s:%s from JSON file via oc-create command", resourceType, resourceName))
+	_, err := ktests.RunOcCommand("create", "-f", filePath)
+	Expect(err).ToNot(HaveOccurred())
+	Eventually(func() bool {
+		out, err := ktests.RunOcCommand("get", resourceType)
+		Expect(err).ToNot(HaveOccurred())
+		return strings.Contains(out, resourceName)
+	}, time.Duration(2)*time.Minute).Should(BeTrue(), "Timed out waiting for ovm to appear")
+}
+
+func WaitUntilResourceReady(resourceName, resourceType, query, expectOut string) {
+	By(fmt.Sprintf("Wait until %s:%s ready", resourceType, resourceName))
+	Eventually(func() bool {
+		//oc get resource with -o jsonpath query
+		out, err := ktests.RunOcCommand("get", resourceType, resourceName, query)
+		Expect(err).ToNot(HaveOccurred())
+		return strings.Contains(out, expectOut)
+	}, time.Duration(2)*time.Minute).Should(BeTrue(), "Timed out waiting for vm to appear")
+}
+
+func writeJson(name string, json string) (string, error) {
+	jsonFile := fmt.Sprintf("%s.json", name)
+	err := ioutil.WriteFile(jsonFile, []byte(json), 0644)
+	if err != nil {
+		return "", fmt.Errorf("failed to write json file %s", jsonFile)
+	}
+	return jsonFile, nil
+}
```

Comment:
Newline please.

Reply:
It's a de-facto standard.  You'll notice github itself flags this as an issue when displaying changes.